### ISSUE TITLE
tidem() "standard" constituents do not match doc

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1180]
This came up in solving #1179. The docs say the "standard" constituents are (in roxygen format)
```R
#' \item Case 1. If \code{constituents} is not provided, then the constituent
#' list will be made up of the 69 constituents regarded by Foreman as standard.
#' These include astronomical frequencies and some shallow-water frequencies,
#' and are as follows: \code{c("Z0", "SA", "SSA", "MSM", "MM", "MSF", "MF",
#' "ALP1", "2Q1", "SIG1", "Q1", "RHO1", "O1", "TAU1", "BET1", "NO1", "CHI1",
#' "PI1", "P1", "S1", "K1", "PSI1", "PHI1", "THE1", "J1", "SO1", "OO1", "UPS1",
#' "OQ2", "EPS2", "2N2", "MU2", "N2", "NU2", "GAM2", "H1", "M2", "H2", "MKS2",
#' "LDA2", "L2", "T2", "S2", "R2", "K2", "MSN2", "ETA2", "MO3", "M3", "SO3",
#' "MK3", "SK3", "MN4", "M4", "SN4", "MS4", "MK4", "S4", "SK4", "2MK5", "2SK5",
#' "2MN6", "M6", "2MS6", "2MK6", "2SM6", "MSK6", "3MK7", "M8")}.
```
but the code is coming out as below (from test code).
```R
> tide1[["data"]]$name
 [1] "Z0"   "SSA"  "MSM"  "MM"   "MSF"  "MF"   "ALP1" "2Q1"  "SIG1" "Q1"   "RHO1"
[12] "O1"   "TAU1" "BET1" "NO1"  "CHI1" "P1"   "K1"   "PHI1" "THE1" "J1"   "SO1" 
[23] "OO1"  "UPS1" "OQ2"  "EPS2" "2N2"  "MU2"  "N2"   "NU2"  "M2"   "MKS2" "LDA2"
[34] "L2"   "S2"   "K2"   "MSN2" "ETA2" "MO3"  "M3"   "SO3"  "MK3"  "SK3"  "MN4" 
[45] "M4"   "SN4"  "MS4"  "MK4"  "S4"   "SK4"  "2MK5" "2SK5" "2MN6" "M6"   "2MS6"
[56] "2MK6" "2SM6" "MSK6" "3MK7" "M8"  
```

**Action item** I'll look in the Foreman documents and check on what should be called "standard". Perhaps there is a selection based on the length of the dataset (but I doubt it).

**Comment by dankelley** [comment]
Below are the missing ones:
```R
> standard[!standard %in% tide1[["data"]]$name]
[1] "SA"   "PI1"  "S1"   "PSI1" "GAM2" "H1"   "H2"   "T2"   "R2"  
```

**Comment by dankelley** [comment]
Oh, I think this is an issue of the elimination of constituents because of the Rayleigh criterion. (One must have a long time-series to distinguish between two frequencies that are very close together.) I am looking at the document [1] . My test is with `data(sealevel)` and it has the following length in hours.
```R
> diff(range(as.numeric(sealevel[['time']]))) / 3600
[1] 6718
```
Now, consider Table 1 (page 19 of the scan). It says if an hourly timeseries is shorter than (or maybe <=, not sure) 8766 hours, constituent `SA` should be dropped.

Next, consider Table 2 (page 20 of the scan). It says that if it's shorter than 8767 hours we drop also `PI`, `S1` and `PSI1`.

Next, consider Table 3 (page 21 of the scan). It says that if it's shorter than 8767 hours we drop also `H1`, `H2`, `T2` and `R2`.

OK, then, that's the whole list of the ones we do not have. I conclude that things are being done properly, but that I need to clarify the documentation. I will close this issue when I've clarified the documentation. (I'll post below the change to the doc ... most likely I will just write a paragraph along the lines of the above few items.)

**References**
1. MANUAL FOR TIDAL HEIGHTS ANALYSIS AND PREDICTION by M.G.G. Foreman (http://www.dfo-mpo.gc.ca/Library/54866.pdf, downloaded 2016-01-24)

**Comment by dankelley** [comment]
Fixed in `develop` commit 9e6c376a54d9991e7e4b24c684472905b4937b04 (and a few commits made earlier than that ... I forgot to put a comment here when I fixed this).
